src/compiler/Stage/Parse/Parser.elm

```diff
@@ -341,13 +350,15 @@ expr =
             -- TODO test this: does `x =\n  call 1\n+ something` work? (it shouldn't: no space before '+')
             [ PP.infixLeft 99
                 checkNotBeginningOfLine
-                (\fn argument ->
-                    Frontend.Call
-                        { fn = fn
-                        , argument = argument
-                        }
+                (Located.merge
+                    (\fn argument ->
+                        Frontend.Call
+                            { fn = fn
+                            , argument = argument
+                            }
+                    )
                 )
-            , PP.infixLeft 1 (P.symbol (P.Token "+" ExpectingPlusOperator)) Plus
+            , PP.infixLeft 1 (P.symbol (P.Token "+" ExpectingPlusOperator)) (Located.merge Plus)
```

Comment:
Here it's merged, but nowhere else... either it shouldn't merge at all, or merge also in function, let, lists..

Reply:
I don't know if that's true.
I think it should merge in all the `andThenOneOf` cases, but not in the `oneOf` cases :thinking: as those don't take a second Expr as an argument, right?

Reply:
I remember why I made it now. I used generated expressions so the tests told me this was right. I was really tired when commented last night ^^.

Reply:
OK, then this code probably doesn't need changing :) marking as resolved.